src/index.html

```diff
@@ -3,11 +3,54 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+
+    <link rel="stylesheet" href="../src/styles/style.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div data-qa="card" class="container">
+      <div class="container__image">
```

Comment:
```suggestion
    <div data-qa="card" class="card">
      <div class="card__image">
```
You can name it 'card' to make it cleaner